From 46e230931e12eea90b4eb903f80ec0dcc57277b2 Mon Sep 17 00:00:00 2001
From: goglepox <goglepox@fcb5ba71-d80d-0410-8237-ba3920747fcc>
Date: Mon, 15 Jun 2009 16:16:29 +0000
Subject: [PATCH] Move atom nowhere

The atom is moved with the ChemModelRelay to get a coordinatesChanged event
[jchempaint-primary] r14615

git-svn-id: https://bioclipse.svn.sourceforge.net/svnroot/bioclipse/bioclipse2/trunk@11363 fcb5ba71-d80d-0410-8237-ba3920747fcc
---
 .../org/openscience/cdk/controller/MoveModule.java |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/cdk-externals/trunk/org.openscience.cdk.control/src/org/openscience/cdk/controller/MoveModule.java b/cdk-externals/trunk/org.openscience.cdk.control/src/org/openscience/cdk/controller/MoveModule.java
index 32a9db2..edc9128 100644
--- a/cdk-externals/trunk/org.openscience.cdk.control/src/org/openscience/cdk/controller/MoveModule.java
+++ b/cdk-externals/trunk/org.openscience.cdk.control/src/org/openscience/cdk/controller/MoveModule.java
@@ -101,7 +101,7 @@ public class MoveModule extends ControllerModuleAdapter {
                 mergeMolecules(end);
             } else {
                 for(IAtom atom:selection.getConnectedAtomContainer().atoms()) {
-                    chemModelRelay.moveTo( atom, worldCoord );
+                    chemModelRelay.moveTo( atom, atom.getPoint2d() );
                 }
                 IUndoRedoFactory factory = chemModelRelay.getUndoRedoFactory();
                 UndoRedoHandler handler = chemModelRelay.getUndoRedoHandler();
-- 
1.6.0.4

